index.d.ts

```diff
@@ -192,6 +193,10 @@ export enum MetricType {
 	Summary,
 }
 
+export type ClusterMetricsOptions = {
+	timeoutMs?: number;
```

Comment:
```suggestion
	/** Timeout to wait for all workers to respond before error. **/
	timeoutMs?: number;
```

This can be documented as well